biz.aQute.bndlib.tests/src/test/component/DSAnnotationTest.java

```diff
@@ -1394,7 +1398,9 @@ public void testBinds(boolean extender) throws Exception {
 		Builder b = new Builder();
 		b.setProperty(Constants.DSANNOTATIONS, "test.component.DSAnnotationTest*CheckBinds");
 		if (extender)
-			b.setProperty(Constants.DSANNOTATIONS_OPTIONS, "extender");
```

Comment:
This would be a good place to test the merge properties support. So instead of an else with just version, leave the if (extender) alone and add:

```java
b.setProperty(Constants.DSANNOTATIONS_OPTIONS+".min", "version;minimum=1.2.0");
```